ztp/source-crs/PtpConfigDualCardGmWpc.yaml

```diff
@@ -115,9 +116,9 @@ spec:
       masterOnly 1
       [$iface_master_1]
       masterOnly 1
-      [$iface_master_1_1]
+      [$iface_master_nic2]
       masterOnly 1
-      [$iface_master_1_2]
+      [$iface_master_nic2_1]
       masterOnly 1
       [global]
```

Comment:
Suggest to add all 4 ports for each NIC. i.e., 8 ports in total.

Reply:
Is 4 ports the most typical setup?

Reply:
added more ports